packages/main/test/pages/DialogAndOpenUI5Dialog.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+	<title>Dialog</title>
+	<script data-ui5-config type="application/json">
+		{
+			"language": "EN"
+		}
+	</script>
+	<script>
+		// delete Document.prototype.adoptedStyleSheets
+	</script>
+	<script src="%VITE_BUNDLE_PATH%" type="module"></script>
+	<script src="https://openui5.hana.ondemand.com/resources/sap-ui-core.js"
+			id="sap-ui-bootstrap"
+			data-sap-ui-libs="sap.m"
+			data-sap-ui-oninit="onOpenUI5Init"
+			data-sap-ui-compatVersion="edge"></script>
+	<script>
+		function onOpenUI5Init() {
+			sap.ui.require(["sap/m/Button"], (Button) => {
+				new Button("openUI5Button", {
+					text: "OpenUI5 Button"
+				}).placeAt("content");
+			});
+		}
+
+		function init() {
+			document.getElementById("myButton").addEventListener("click", function() {
+				document.getElementById("dialog1").open = true;
+			});
+
+			document.getElementById("dialogButton").addEventListener("click", function () {
+				sap.ui.require(["sap/m/Button", "sap/m/Dialog"], (Button, Dialog) => {
+					new Dialog({
+						title: "OpenUI5 Dialog",
+						content: new Button("openUI5DialogButton", {
```

Comment:
add another focusable element before the button so we can check that focus goes back to correct position after closing the ui5-dialog